packages/conf-graphics/conf-graphics.1.0/opam

```diff
@@ -0,0 +1,8 @@
+opam-version: "1"
+maintainer: "Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"
+homepage: "http://www.ocaml.org/"
+license: "LGPL+EXN-BIN"
+build: [["ocamlc" "-custom" "graphics.cma" "-o" "test"]]
+post-messages: [
+  "This package checks whether the Graphics library was compiled." {failure}
```

Comment:
some hints in here about how to install it? e.g. the brew install for xquartz here would be a common failure i suspect https://medium.com/partially-applied/stubbed-toes-getting-started-with-ocaml-on-os-x-9ff16768a12d#.bjg7h0wio